app/templates/custom-elements/remote-screen.html

```diff
@@ -158,34 +158,11 @@
           this.shadowRoot.addEventListener("touchend", () => {
```

Comment:
When I tested this on Android + Firefox, it treats separate touch events as a mouse button that's being held down:

https://github.com/tiny-pilot/tinypilot/assets/7783288/cab78f08-6f53-4ea4-a6cd-718d8c101cbb

Reply:
Looks like mouseup event is not being fired on firefox, I'll debug this tonight.

Reply:
Fixed

Reply:
Cool, good progress! I still can't click and drag on Android. Is that expected?

https://github.com/tiny-pilot/tinypilot/assets/7783288/e0e90f0b-0152-415a-871f-303eb08461fb

Reply:
I forgot that functionality, I'll try to add it in a new PR.